AnisimovAN/lab_4/lab_4.cpp

```diff
@@ -0,0 +1,143 @@
+#include "pch.h"
+#include <iostream>
+#include <conio.h>
+
+
+using namespace std;
+
+double power(double number_stepen, int stepen)
+{
+	double rez = 1;
+
+	if (stepen > 0) {
+		for (int i = 0; i < stepen; i++)
+		{
+			rez *= number_stepen;
+		}
+	}
+	else
+	{
+		for (int i = 0; i < (-1) * stepen; i++)
+		{
+			rez = rez * number_stepen;
+		}
+		rez = 1 / rez;
+	}
+
+	return rez;
+}
+
+int fact(int number_fact)
+{
+	int rez = 1;
+
+	for (int i = 1; i <= number_fact; i++)
+	{
+		rez *= i;
+	}
+
+	return rez;
+}
+
+double sqrt(int number_sqrt)

```

Comment:
This function looks suspiciously similar to #169 
Was it founded on the internet or it's a result of collective brainstorm?

Reply:
It's my recomend to Ivan Vazhov to use Nyoton Method, I didn't give him any links or files. Just name of math termin.